### DISCUSSION TITLE
FNF Megapack : Bob's Takeover Update!

### DISCUSSION THREAD

**Discussion by realDunkin** [realDunkin/fnfmegapack#33] [Announcements]
# FNF Megapack : Bob's Takeover Update! 

### Patch Notes:
- Added Credits!!!
- Removed HIFI and LOFI Songs
- Added Rate Changer (Built in Speed Modifier)
- Added Psych Engine Song Timer and Score Effect
- Updated Shaggy Mod
- Added Matt Wiik3.
- Added Bob & Bosip Mod.
- Added Bob's Onslaught Mod.
- Added Viernes Noche Webiando
- Added Accelerant to Tricky Mod Category
- Added 5K,7K,8K
- Added 5K,7K,8K Playable Support for Android
- Changed Keybind Menu to separate 3,6,9 key controls into different sections.
- Added more new menu music.
- Fixed Modifiers not saving/loading correctly
- Added Custom Scroll Speed!
- Changed the Category Menu

- Added Mods folder!
- Replace a song or add a new one!
- Replace music / sfx
- Images don't work yet!

- ADDED ONLINE LEADERBOARDS
- Added Accounts (You have to sign in to be on leaderboards)
- You can delete your own scores from leaderboard!
- Some songs don't have leaderboards yet, be patient!
- NOTE: Source code wont have leaderboards working, only release versions.

- Fixed songs with special arrows
- Fixed Slow Songs Repeating Itself, But they a bit laggy.

### Known Bugs:
- Lower half of Expurgation stage is missing (Will Appear After Loaded Once???).
- Release won't end
- HTML5 crashes when trying to sign in
- HTML5 has a hard time trying to get leaderboard stats / account data. 

### Downloads:
- Source Code (Bob's Takeover Update):
https://mega.nz/file/YcR1CQ4a#p9jX272FTCPoOl3lmlrqkVNpSgDxvAlOpHuyRk8fVOM
- Windows 64 Bit (Bob's Takeover Update):
https://mega.nz/file/VQpkjCza#QRxVFNTk-40wMZjZUKYV2z2O1CdH61bV5q5Rmz5DwdQ
- Android (Bob's Takeover Update):
https://mega.nz/file/pZg1zQ7T#j94DUmDKQGjBpL1AmTYVdBr7PvRGqEGRptDLroCWK_U
- HTML5 Version (Bob's Takeover Update):
https://realdunkin.github.io/fnfmegapack-html5/
- Windows 32 Bit: Can't Compile Yet
- MacOS: Can't Compile Yet
- Linux: Can't Compile Yet

**Comment by saragannett** [comment]
anyone have any links to fnf mods any will do!

**Comment by saragannett** [comment]
one that you can play with no download
